manifests/plugin/openscap/params.pp

```diff
@@ -0,0 +1,7 @@
+# Default parameters for the Openscap smart proxy plugin
+class foreman_proxy::plugin::openscap::params {
+  $enabled                 = true
```

Comment:
That's because we switched the design of #140 around a couple of times and I might have had some older version in my head. You're right that now it can just be ```true```.